shuffle/cli/tutorials/HelloBlockchain.md

```diff
@@ -0,0 +1,164 @@
+```
+
+## Run a Node
+
+Run `shuffle node` to deploy a local test node.
+If you are testing against a remote node, you can skip this step.
+
+Output:
+
+```
+Creating node config in /Users/sunmilee/.shuffle
+Completed generating configuration:
+	Log file: "/Users/sunmilee/.shuffle/nodeconfig/validator.log"
+	Config path: "/Users/sunmilee/.shuffle/nodeconfig/0/node.yaml"
+	Diem root key path: "/Users/sunmilee/.shuffle/nodeconfig/mint.key"
+	Waypoint: 0:4adac71a66fe1725c53735156d7b4467bf52915c082f00d4db3cb980135335c7
+	ChainId: TESTING
+	JSON-RPC endpoint: 0.0.0.0:8080
+	REST API endpoint: 0.0.0.0:8080
+	Stream-RPC enabled!
+	FullNode network: /ip4/0.0.0.0/tcp/7180
+
+	Lazy mode is enabled
+
+Diem is running, press ctrl-c to exit
+```
+
+## Create an Account
+In order to deploy your modules, you must have an account on-chain. Run `shuffle account` to create  accounts on the default localhost network.
+
+Output:
+
+```
+Connecting to http://0.0.0.0:8080...
+Creating a new account onchain...
+Successfully created account 24163AFCC6E33B0A9473852E18327FA9
+Private key: 2f17c7a5f1175945aceb430f97e6413a89ce8f951e668283153c8f9f7e413bb6
+Public key: a831c99b7a599965b716eb35b33d813342f028cdb5f0f0c068d7172055ecfff5
+Creating a new account onchain...
+Successfully created account D770294D6EEAB56836AB02735083E4EC
+Private key: 6204190004b94c89dbfc993c18f91344fb7f5058bb9ce8e9a0ab79dac9d6592f
+Public key: b1ca8b33235bf1f01b3e9d0278555858acb0a2d7cc1c2f428643fff18ff0e5ce
+```
+
+## Write a Module
+Now you are ready to write your own modules. To start, create a move file in `project_directory/main/sources`.
+There is an example module `Message.move` that sets a message to `MessageHolder` move resource that you can play with.
+
+## Deploy Your Module
+
+Once you have your move module, you will need to deploy your module on-chain using a transaction.
+`cd` into your project directory and run `shuffle deploy`.
+This will compile all move modules inside the `helloblockchain/main` directory and deploy them on-chain.
+
+Output:
+```
+Using Public Key a831c99b7a599965b716eb35b33d813342f028cdb5f0f0c068d7172055ecfff5
+Sending txn from address 0x24163afcc6e33b0a9473852e18327fa9
+Building /Users/sunmilee/diem/helloblockchain/main...
+CACHED MoveStdlib
+CACHED Message
+Skipping Module: 00000000000000000000000000000001::Hash
+Skipping Module: 00000000000000000000000000000001::Signer
+Skipping Module: 00000000000000000000000000000001::Vector
+Skipping Module: 00000000000000000000000000000001::Errors
+Skipping Module: 00000000000000000000000000000001::BitVector
+Skipping Module: 00000000000000000000000000000001::Capability
+Skipping Module: 00000000000000000000000000000001::FixedPoint32
+Skipping Module: 00000000000000000000000000000001::Option
+Skipping Module: 00000000000000000000000000000001::BCS
+Skipping Module: 00000000000000000000000000000001::GUID
+Skipping Module: 00000000000000000000000000000001::Event
+Deploying Module: 24163AFCC6E33B0A9473852E18327FA9::Message
+```
```

Comment:
Did we purposefully leave the NFT/TestNFTs out of this output?

Reply:
yep